config/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	OTExporterOTLPEndpoint     string        `envconfig:"OTEL_EXPORTER_OTLP_ENDPOINT"`
 	OTServiceName              string        `envconfig:"OTEL_SERVICE_NAME"`
 	BabbageURL                 string        `envconfig:"BABBAGE_URL"`
+	RelCalURL                  string        `envconfig:"RELEASE_CALENDAR_URL"`
```

Comment:
very small - but as we're using `BabbageURL`, `LegacyCacheAPIURL` etc perhaps `ReleaseCalendarURL` is more appropriate?

Reply:
This is how it's been named within other repos so we though we'd keep it consisted with those